documentation/v1/docs.css

```diff
@@ -44,7 +44,6 @@ li {
 }
 table {
   margin: 16px 0 0 13px; padding: 0;
-  width: 690px;
```

Comment:
This was stretching out the table of operators, which is why I took it out. The table of usage flags is now just the width of the code examples: http://rawgit.com/geoffreybooth/coffeescript/1.12.5/docs/v1/index.html